GD32F4XX/standard_peripheral/Source/gd32f4xx_can.c

```diff
@@ -38,6 +38,15 @@ OF SUCH DAMAGE.
 
 #include "gd32f4xx_can.h"
 
+/* CAN communication mode */
+#define CAN_NORMAL_MODE                    ((uint8_t)0x00U)             /*!< normal communication mode */
+#define CAN_LOOPBACK_MODE                  ((uint8_t)0x01U)             /*!< loopback communication mode */
+#define CAN_SILENT_MODE                    ((uint8_t)0x02U)             /*!< silent communication mode */
+#define CAN_SILENT_LOOPBACK_MODE           ((uint8_t)0x03U)             /*!< loopback and silent communication mode */
```

Comment:
This change prevents user to initialize `can_parameter_struct` `working_mode`, so just keep them in the header file by prefixing them, e.g. `__CAN_NORMAL_MODE`.

Reply:
Done.